nslsii/areadetector/xspress3.py

```diff
@@ -135,7 +148,7 @@ def describe(self):
             dict(
                 external="FILESTORE:",
                 dtype="array",
-                dtype_str="uint32",  # <u4
+                dtype_str=self.dtype_str,
```

Comment:
I remember for CHX the valid values for `dtype_str` were specified. Should we do this validation here too?

https://github.com/NSLS-II-CHX/profile_collection/blob/a33842a933d2db656b1f91178e7bb5ad8fdf239b/startup/20-area-detectors.py#L46-L49

Reply:
Once we know the valid values....